tiledb/sm/serialization/array.h

```diff
@@ -58,7 +58,7 @@ namespace serialization {
  * Serialize an Array to Cap'n proto
  * @param array to serialize
  * @param array_builder cap'n proto class
- * @param cleint_side is serialization client or server side
+ * @param client_side is serialization client or server side
```

Comment:
i still don't know what this _means_ -- somewhere we need a one-sentence something about what a 'client-side perspective' is ...

Reply:
Thank you for the comments!
Added more detailed info, please let me know if that's still unclear.